### ISSUE TITLE
🐛 (RC):  Tags can only be added inconsistently to "Tag involved in this release control" field

### ISSUE THREAD

**Issue by EDUT-Equinor** [equinor/lighthouse-client#1533]
Error
## Description

 
This seems to be a temperamental issue, and only some tags are affected.

Sometimes, some tags in the "Tag involved in this release control" field will not save themselves into the list of tags. e.g., with 50FO0050. It is a soft fail so users may not realize a tag is not added.
 
![image](https://github.com/equinor/lighthouse-client/assets/147052136/d735f704-2a2b-4735-b143-6b7955d5a800)

pics of my F12 tab: no1
![image](https://github.com/equinor/lighthouse-client/assets/147052136/08744006-6822-4986-860b-1066cdd90c75)

no2:
![image](https://github.com/equinor/lighthouse-client/assets/147052136/73547654-c6d5-496c-a07a-cc415059e3b5)

no3
![image](https://github.com/equinor/lighthouse-client/assets/147052136/3d629943-fa9c-41f5-9be5-178d5720a5fc)

## Reproduce

try to add a tag, e.g, 50FO0050, and observe that it does not get added to the list as expected. 

## Expected behavior

 
- [ ] all tags can be added

**Comment by EDUT-Equinor** [comment]
@idrisbo  - ah no idea why it is only sometimes happening, I just marked it as blocked and we come back to it when it happened again.  I am not getting an error now. Perhaps it was incidently fixed with another PR or something like that!

**Comment by ken-mellem** [comment]
I prefer closing as `cannot reproduce`. The label `blocked` is used to indicate the issue is actually being blocked by another issue or external factor/dependency. You can search for them by using the filter `reason:"not planned"`

**Comment by MagnusGjosund** [comment]
Happened again. This time with tag: vg53-0032

**Comment by EDUT-Equinor** [comment]
Maybe connected to this bug. Tag below i could not add in a similar way that is described above. after 1 hour (and submitting the request) i was able to add the tag, but now the tag appears twice.
![Image](https://github.com/equinor/lighthouse-client/assets/147052136/7eca1e8b-d29e-41eb-a0b3-aeefe4b84bd6)

**Comment by Gustav-Eikaas** [comment]
The two objects in the response from fam has identical guid's indicating this is a sync issue in fam.

**Comment by ken-mellem** [comment]
> The two objects in the response from fam has identical guid's indicating this is a sync issue in fam.
 
Have you notified FAM?
